.circleci/config.yml

```diff
@@ -247,6 +247,12 @@ jobs:
             mkdir -p artifacts-<< parameters.platform >>/gems
             mv pkg/*.gem artifacts-<< parameters.platform >>/gems
 
+      - run:
+          name: Remove non-native gem to avoid conflict in workspace
```

Comment:
Does this mean there's unexpected shared state between the CI jobs? Addressing that might be a better approach than "delete everything that's not me" 🤔

Reply:
It is a rather undesired side-effect of `gem build`: Each build yields its non-native version of the gem, on which CircleCI reports a conflict of resources. I did some research and there seems to be not a way to surpress the generation of the non-native gem.